src/Tizen.Sensor/Tizen.Sensor/EventArgs/ProviderIntervalChangedEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Sensor
+{
+    /// <summary>
+    /// Indicates Provider Interval Changed callback.
+    /// </summary>
+    public class ProviderIntervalChangedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Indicates Provider Interval Changed callback.
+        /// </summary>
+        internal ProviderIntervalChangedEventArgs(IntPtr _userData, uint intervalMs) {
+            IntervalMs = intervalMs;
+            UserData = _userData;
+        }
+
+        /// <summary>
+        /// Interval
+        /// </summary>
+        public uint IntervalMs { get; private set; }
+        /// <summary>
+        /// Indicates userdata sent by provider Interval changed callback
+        /// </summary>
+        public IntPtr UserData { get; private set; }
```

Comment:
User shouldn't have access to IntPtr. Please wrap it up to some SafeHandle or redesign this class.